src/components/data-collection-page/DataCollectionPage.tsx

```diff
@@ -802,7 +465,7 @@ export default function DataCollectionPage(props: IProps) {
 						style={{ gap: '10px' }}
 					>
 						<ToggleButton
-							value={ Climb.none }
+							value='None'
```

Comment:
The `value`s for each of these buttons and those of the Park toggles should remain enum values, not strings